lib/actions-oclif/build.ts

```diff
@@ -0,0 +1,273 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../command';
+import * as cf from '../utils/common-flags';
+import { getBalenaSdk } from '../utils/lazy';
+import * as compose from '../utils/compose';
+import DockerToolbelt = require('docker-toolbelt'); // https://github.com/balena-io-modules/docker-toolbelt/pull/40
+import Logger = require('../utils/logger');
```

Comment:
Any idea how to remove the need for this line?  We just need the type imported.  The logger import is now done dynamically in the base class. (See below)

Reply:
To make it a type-only import here you'd to change the logger export to be es module compatible so you can then do eg
```
import type { Logger } from '../utils/logger'
```

Reply:
Yeah I did change the logger module to have the class a s default export, which fixes that problem, but then breaks a load of other stuff. Maybe best to try to do that in a separate PR.